src/cryptenroll/cryptenroll-pkcs11.c

```diff
@@ -1,6 +1,13 @@
-/* SPDX-License-Identifier: LGPL-2.1-or-later */
+/* SPDX-License-Identifier: LGPL-2.1-or-later
+ *
+ * Copyright © 2024 GNOME Foundation Inc
+ *      Original Author: Adrian Vovk
```

Comment:
original author is a bit much for a file that is pre-existing, no?

i mean, generally we don#t really do this. we tell people to check git history/blame to see who contributed what.

is the gnome foundation asking you to add this?

Reply:
I suppose I can drop the original author bit but the GNOME foundation wants me to put a copyright header. I just tend to add it because I'm allowed to and I want the work to be attributed to me

I definitely didn't intend to say the whole file was originally authored by me... I could either qualify it like this:

```
Varlink Server Implementation
Copyright © 2024 GNOME Foundation Inc.
     Original Author: Adrian Vovk
```

Or I suppose I can just drop the "Original Author" bit and let `git blame` handle it

Reply:
We add all authors who contribute to a given release in the NEWS file, we could list the foundation there too explicitly? Would that satisfy the requirement?